apps/search/api/filters/elastic_filters.py

```diff
@@ -36,7 +36,9 @@ def get_filter_params(request):
         indexes = '*'
         content_types = request.query_params.getlist('content_types') or None
         if content_types:
-            indexes = ','.join(content_types)
+            # #ContentTypesHardCoded
```

Comment:
I don't know if this is a pattern we want to start using, but using "linked comments" like this was common at Asana to indicate when there is strong coupling to another part of the codebase.

Reply:
No strong opinion